docs/new_instrument.rst

```diff
@@ -601,6 +609,7 @@ generally be put in the `init` function of each instrument.
 
         self.acknowledgements = acknowledgements_string
         self.references = references_string
+	logger.info(self.acknowledgements)
```

Comment:
tab not aligned

Reply:
Suggestion forthcoming?

Reply:
Eh, nevermind, I am going to go into this properly and fix the load routine too.